cpp/src/link_analysis/pagerank_impl.cuh

```diff
@@ -151,8 +140,8 @@ void pagerank(
         pull_graph_view.local_vertex_partition_view());
       auto num_invalid_vertices =
         thrust::count_if(handle.get_thrust_policy(),
-                         *personalization_vertices,
-                         *personalization_vertices + *personalization_vector_size,
+                         std::get<0>(*personalization).begin(),
+                         std::get<0>(*personalization).end(),
                          [vertex_partition] __device__(auto val) {
                            return !(vertex_partition.is_valid_vertex(val) &&
                                     vertex_partition.in_local_vertex_partition_range_nocheck(val));
```

Comment:
Should there be an additional check to make sure that the vertices provided in `personalization` are unique?